huaweicloud/services/acceptance/cnad/resource_huaweicloud_cnad_advanced_black_white_list_test.go

```diff
@@ -16,11 +16,12 @@ import (
 )
 
 func getBlackWhiteListResourceFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	region := acceptance.HW_REGION_NAME
 	var (
 		getBlackWhiteHttpUrl = "v1/cnad/policies/{policy_id}"
```

Comment:
suggest to use unified format to define variables, please check others and fix them
```suggestion
	var (
		region = acceptance.HW_REGION_NAME
		getBlackWhiteHttpUrl = "v1/cnad/policies/{policy_id}"
```

Reply:
done